### PR TITLE
unify database parameters

### PR DESCRIPTION
**Pull request by mmoll**
see also rurseekatze/node-tileserver#140

### CODE REVIEW

**Comment by DerDakon** [comment]
The prefix in api/config.php needs to be changed, too.

**Comment by Nakaner** [comment]
@rurseekatze Did you change the name of the database when you set it up on the new sever a few weeks ago?

File: api/config.php [hunk 1]
```diff
@@ -51,9 +51,9 @@
 		"vi" => array("vi_VN", "Tiếng Việt")
 	);
 	// name of database
-	$db = "railmap";
+	$db = "openrailwaymap";
 	// prefix of osm2pgsql tables
-	$prefix = "railmap";
+	$prefix = "openrailwaymap";
```
Comment (mmoll) [inline api/config.php]: this prefix?

Comment (mmoll) [inline api/config.php]: @DerDakon this prefix?

Comment (DerDakon) [inline api/config.php]: yes

**Pull request merged**